src/analysis/convex/basic.lean

```diff
@@ -7,6 +7,8 @@ import data.set.intervals.ord_connected
 import data.set.intervals.image_preimage
 import data.complex.module
 import linear_algebra.affine_space.basic
+import algebra.module.ordered
+
 
 /-!
 # Convex sets and functions on real vector spaces
```

Comment:
You should add something in the module doc about this generalisation. (Perhaps a note at the end, rather than writing all the text in terms of the general situation.)

Reply:
Done!